pytket/docs/index.rst

```diff
@@ -84,12 +84,13 @@ Licensed under the `Apache 2 License <http://www.apache.org/licenses/LICENSE-2.0
     faqs.rst
 
 .. toctree::
-    :caption: More Documentation:
+    :caption: pytket documentation:
     
-    TKET website <https://tket.quantinuum.com/>
-    Manual <https://tket.quantinuum.com/user-manual/>
-    extensions.rst  
+    pytket API docs <https://tket.quantinuum.com/api-docs/>
+    extensions.rst
+    Manual <https://tket.quantinuum.com/user-manual>
     Example notebooks <https://tket.quantinuum.com/examples>
+    TKET website <https://tket.quantinuum.com/>
```

Comment:
Because of some merge conflicts I am not sure if this is the right order of the links?

Reply:
The order looks right to me

Reply:
This is the order we have here, so should be fine :)

I would assume that we are missing the maxdepth on purpose for the extension list?

Reply:
Will double check this quickly

Reply:
I double checked and the docs look the same if the `:maxdepth` is set or not.